apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisStackedColumnChart.vue

```diff
@@ -98,7 +98,7 @@ const drawChart = () => {
     yAxis.get('renderer').labels.template.adapters.add('text', (text) => {
         if (text) {
             const convertedText = text.replace(/,/g, '');
-            return commaFormatter(numberFormatter(Number(convertedText)));
+            return commaFormatter(numberFormatter(Number(convertedText), 1, 0));
```

Comment:
suggestion: I'm concerned because formatters are being used nested. It seems like I need to organize things related to formatters at once. 😓